src/documentation/reference.md

```diff
@@ -6,6 +6,6 @@ This page shows all the features provided by the library.
 
 The `doNothing` function takes anything as parameter and returns it unchanged:
 
-@@snip [Usage.scala]($root$/src/test/scala/ch/epfl/scala/Usage.scala) { #do-nothing }
+@@snip [Usage.scala](./../../src/test/scala/ch/epfl/scala/Usage.scala) { #do-nothing }
```

Comment:
Unfortunately the path is relative to `Paradox / sourceDirectory` therefore to `mdocOut` which does not contain the sources. Any idea on how to improve?

Reply:
I’m confused because it seems your changes are unrelated to this. Did you check that it was working before?

Reply:
`$root$` is supposed to point to the root directory, not subdirectory :confused: 

https://developer.lightbend.com/docs/paradox/current/directives/snippets.html

Reply:
I cannot test if it worked before because `master` uses `tut` which I think it is not published anymore since I get a
```
[error] (update) sbt.librarymanagement.ResolveException: unresolved dependency: org.tpolecat#tut-core_2.12;0.6.10: not found
```